README.md

```diff
@@ -17,10 +17,6 @@ This means that `runc` 1.0.0 should implement the 1.0 version of the specificati
 
 You can find official releases of `runc` on the [release](https://github.com/opencontainers/runc/releases) page.
 
-Currently, the following features are not considered to be production-ready:
-
-* [Support for cgroup v2](./docs/cgroup-v2.md)
```

Comment:
Maybe it makes sense to have a section named "Additional documentation" and then link whatever we have in docs/ from there?

Reply:
Scratch that. It's better to just add a link to the doc (see the comment above)

Reply:
We already have a link in the "cgroup v2" section of the top-level README.md

Reply:
yes we do; right